### ISSUE TITLE
Jetson Nano retraining SSD MobileNet losses are not decreasing

### ISSUE THREAD

**Issue by Lacika180** [dusty-nv/jetson-inference#1224]
Hello everyone,

My project is a road sign (STOP, Give Priority, pedestrian crossings) and pedestrian recognizing system. I followed @dusty-nv 's tutorial about training custom object detection model with transfer learning on Jetson Nano (SSD MobileNet v1 300x300). I gathered 800 images, labeled them with the CVAT and separated them into 3 parts (70% train, 20% validation and 10% test). The training process successfully begins, but the main problem is that the loss values are not decreasing after after I reach ~3.5 total loss. I tried using other learning rates as 0.01, 0.001 and 0.0001 but it didn't help. I left the training process for 700 epochs but the smallest loss that I have achieved is 3.17. 
What should I do in order to reach a loss around 0.2? I'm thinking in training with Tensorflow but that program is a little complicated and after training if I know well I have to convert it to ONNX format then in TRT which seems so problematic to me.

Thank you in advance for your help!
